index.html

```diff
@@ -1,6 +1,6 @@
 <!-- This file will be run from ./.tmp/, so make paths relative to there -->
 <!DOCTYPE html>
-<html id="ov1">
+<html id="ov1" class="pre-release"><!-- remove pre-release class when released -->
```

Comment:
I've added the pre-release class here to make it easy to test what things look like if the feedback button and milestone warning aren't present.